charts/shoot-core/components/values.yaml

```diff
@@ -49,7 +49,7 @@ shoot-info:
   maintenanceEnd: 220000+0000
   extensions: shoot-dns-service,foo-bar
 vpn-shoot:
-  enabled: true
+  enabled: false
```

Comment:
VPN-shoot should still be "the default"

```suggestion
  enabled: true
```

Reply:
but then we would have to explicitly turn it off in code when the featuregate is on.

Reply:
Hm okay, right. Fine then, thanks.